lazer/contracts/solana/scripts/verify_ed25519_message.ts

```diff
@@ -0,0 +1,64 @@
+import * as anchor from "@coral-xyz/anchor";
+import { Program } from "@coral-xyz/anchor";
+import { PythLazerSolanaContract } from "../target/types/pyth_lazer_solana_contract";
+import * as pythLazerSolanaContractIdl from "../target/idl/pyth_lazer_solana_contract.json";
+import yargs from "yargs/yargs";
+import { readFileSync } from "fs";
+import NodeWallet from "@coral-xyz/anchor/dist/cjs/nodewallet";
+import { createEd25519Instruction } from "@pythnetwork/pyth-lazer-sdk";
```

Comment:
actually can you move this function to `@pythnetwork/pyth-lazer-solana-contract`. It makes sense to be here and we want to remove it from js-sdk to remove the solana dependencies.

Reply:
I removed solana deps from `@pythnetwork/pyth-lazer-sdk` and moved the code to ``@pythnetwork/pyth-lazer-solana-contract`. Preparing `@pythnetwork/pyth-lazer-solana-contract` for publishing is tricky, so I'll leave it for later.